### PR TITLE
Update CartController responses to use proper servlet responses

### PR DESCRIPTION
**Pull request by joshsadam**

### CODE REVIEW

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java [hunk 1]
```diff
@@ -211,13 +211,11 @@ public AddToCartResponse addSamplesToCart(@RequestBody AddToCartRequest addToCar
 	 *            The {@link Project} ID
 	 * @param sampleIds
 	 *            The {@link Sample} ID
-	 * @return a map stating success
 	 */
 	@RequestMapping(value = "/project/{projectId}/samples", method = RequestMethod.DELETE, consumes = MediaType.APPLICATION_JSON_VALUE)
 	@ResponseBody
```
Comment (apetkau) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java]: Does `@ResponseBody` do anything when the return type is `void`?

Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java]: Looks like you don't.  Removed in 4fc8d442ef68e3a6d778777f9c44454ef20b8210

File: src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java [hunk 2]
```diff
@@ -245,7 +239,6 @@ public AddToCartResponse addSamplesToCart(@RequestBody AddToCartRequest addToCar
 	 * @return a map stating success
 	 */
 	@RequestMapping(value = "/project/{projectId}", method = RequestMethod.POST, consumes = MediaType.APPLICATION_JSON_VALUE)
-	@ResponseBody
```
Comment (apetkau) [inline src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java]: This method still returns something does it not?  Should the `@ResponseBody` still be here?

Comment (joshsadam) [reply src/main/java/ca/corefacility/bioinformatics/irida/ria/web/cart/CartController.java]: This method actually does not need to return anything like the others.  Changed the return type to void and removed return statement in 420756e43e614bb700c02b4fdb79a90d343ec139

**Comment by joshsadam** [comment]
@apetkau any other comments on this merge?

**Comment by apetkau** [comment]
Nope. This looks good to me.

**Pull request merged**